op-devstack/dsl/l2_el.go

```diff
@@ -83,11 +83,10 @@ func (el *L2ELNode) AdvancedFn(label eth.BlockLabel, block uint64) CheckFunc {
 	}
 }
 
-func (el *L2ELNode) NotAdvancedFn(label eth.BlockLabel) CheckFunc {
+func (el *L2ELNode) NotAdvancedFn(label eth.BlockLabel, attempts int) CheckFunc {
```

Comment:
It would be better to use the options pattern so that these are optional arguments rather than requiring them everywhere.  The vast majority of cases should just use a sensible default (which could then be easily adjusted if needed) and then the few places that need a custom number of attempts can specify that with something like `WithAttempts(10)`.